nautobot/circuits/templates/circuits/inc/circuit_termination.html

```diff
@@ -2,142 +2,10 @@
 
```

Comment:
Add a `3.0 TODO:` comment here to recommend removing this file entirely in 3.0 - it's not used in the UI as of this code drop, but retained for now in case some App is using it for something.